### ISSUE TITLE
Any format other than .RAW results in an error when I run LFQ analysis on MaxQuant.

### ISSUE THREAD

**Issue by Amir660066** [cox-labs/CoxLab_Bug_Reporting#84]
**Describe the bug**
I exported my data from Bruker in the mzXML format. However, when I load it into MaxQuant for LFQ analysis, I encounter an error in the status column, and the program shuts down. I would appreciate your assistance with this issue.

![error](https://github.com/cox-labs/CoxLab_Bug_Reporting/assets/93537745/3261eab3-d856-4da2-bf1b-bc5d80185548)

**Comment by JurgenCox** [comment]
You need to use the original raw data.

**Comment by Amir660066** [comment]
> You need to use the original raw data.

The raw data from Bruker comes in .d folders, but when I load these folders into MaxQuant, it shows an error stating that the software cannot find a .RAW file in the folder. How can I resolve this issue?

**Comment by Amir660066** [comment]
Dear all,

I am writing to explain my problem and would greatly appreciate any small hints that might help alleviate my desperation. I am relatively new to proteomics, and our laboratory has an older mass spectrometer, the Bruker HCT series with an ion trap. The raw data I receive from this system is not recognized by MaxQuant, as I mentioned previously. I contacted Bruker support, but they were unable to resolve my issue. I have tried converting the raw data, but there has been no improvement in data evaluation.

I would be grateful for any insights on whether this issue can be resolved by developing a new method on our mass spectrometer, or if data generated by such an older system is simply not compatible with software like MaxQuant. I have observed that data generated by newer systems, such as the timsTOF, produce TDF files that are recognized by MaxQuant.

Your assistance would be highly appreciated.

Best regards,
Amir
